tests/foreman/cli/test_discoveryrule.py

```diff
@@ -172,15 +171,34 @@ def test_positive_create_with_org_loc_id(
         assert class_org.name in rule.organizations
         assert class_location.name in rule.locations
 
+        new_org = Box(target_sat.cli_factory.make_org())
+        new_loc = Box(target_sat.cli_factory.make_location())
+        new_hostgroup = Box(
+            target_sat.cli_factory.make_hostgroup(
+                {'organization-ids': new_org.id, 'location-ids': new_loc.id}
+            )
+        )
+        target_sat.cli.DiscoveryRule.update(
+            {
+                'id': rule.id,
+                'organization-ids': new_org.id,
+                'location-ids': new_loc.id,
+                'hostgroup-id': new_hostgroup.id,
+            }
+        )
+        rule = Box(target_sat.cli.DiscoveryRule.info({'id': rule.id}))
+        assert new_org.name in rule.organizations
+        assert new_loc.name in rule.locations
```

Comment:
Can also add an assertion here for the hostgroup we updated with, else there's no reason to update with a hostgroup.

Reply:
Same in `test_positive_create_and_update_with_org_loc_name`

Reply:
Removed hg as it was irrelevant